habitat/plan.sh

```diff
@@ -14,7 +14,6 @@ pkg_deps=(
   core/libxslt
   core/ruby
   core/net-tools
-  core/hab
```

Comment:
Yeah, I can totally see how this is going to cause problems if we leave it in here, and profiles depend on it... especially depending on the workstation that build the InSpec package itself. I'm :+1: removing this and immediately rebuilding the InSpec package on the depot.